packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -1,6 +1,39 @@
 ---
 title: Okta API Products Release Notes 2021
 ---
+
+## October
+
+### Monthly Release 2021.10.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [AES-GCM encryption support for SAML assertions](#aes-gcm-encryption-support-for-saml-assertions) | October 6, 2021 |
+| [Authentication API returns UD User Profile locale is GA in Preview](#authentication-api-returns-ud-user-profile-locale-is-ga-in-preview) | October 6, 2021 |
+| [Native SSO support is GA in Preview](#native-sso-support-is-ga-in-preview) | October 6, 2021 |
+| [Brands API option to remove sign-in page footer message](#brands-api-option-to-remove-sign-in-page-footer-message) | October 6, 2021 |
+| [Bug fixed in 2021.10.0](#bug-fixed-in-2021-10-0)                          | October 6, 2021          |
+
+#### AES-GCM encryption support for SAML assertions
+
+To secure SAML assertions from attacks and to adopt a stronger security mechanism, Okta now supports AES128-GCM and AES256-GCM encryptions modes in addition to AES-128 and AES-256 for SAML applications. <!--OKTA-428056-->
+
+#### Authentication API returns UD User Profile locale is GA in Preview
+
+The response for the primary Authentication API includes the [User Profile's locale](/docs/reference/api/authn/#user-profile-object) value that is the same as the User Profile locale value in Universal Directory (UD). This behavior is now Generally Available in Preview. <!--OKTA-430697-->
```

Comment:
don't need to repeat the GA in Preview part in the content - it's in the title
```suggestion
The response for the primary Authentication API includes the [User Profile's locale](/docs/reference/api/authn/#user-profile-object) value that is the same as the User Profile locale value in Universal Directory (UD). <!--OKTA-430697-->
```

Reply:
You're right, it is redundant. But I thought this our convention (https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/2066710848/API+Release+Note+-+Blurb+Templates)? We need to say "GA in Preview', then repeat the full word... "Generally Available in Preview"?